client/app/telecom/telephony/service/fax/password/telecom-telephony-service-fax-password.controller.js

```diff
@@ -23,7 +23,7 @@ angular.module("managerApp").controller("TelecomTelephonyServiceFaxPasswordCtrl"
             self.passwordForm.isSuccess = true;
             $timeout(function () {
                 self.reset();
-                form.$setPristine();
+                form.$setUntouched();
```

Comment:
What's the impact of not having a pristine form instead of a untouched one?

Reply:
The HTML form event binded to show error message is $touched so setPristine did not reset error messages on field after validation, setUntouched does.